home-page

```diff
@@ -0,0 +1,30 @@
+<!DOCTYPE html>
+<html>
+<head>
+    <title>Binabox NFT Collection</title>
+</head>
+<body>
+
+    <h2>Binabox</h2>
+    <p><a href="#">Home</a></p>
+    <p><a href="#">About</a></p>
+    <p><a href="#">Roadmap</a></p>
+    <p><a href="#">Collection</a></p>
+    <p><a href="#">FAQs</a></p>
+    <p><a href="#">Pages</a></p>
```

Comment:
We have a special tag that says that navigation is here. Will you try to find it and use it for navigation?

Reply:
<nav>  </nav>, this one?

Reply:
nav and the /nav?

Reply:
Yes, you're absolutely right!

Reply:
And you can delete tag `<p>` , it not necessary here